composer.json

```diff
@@ -46,7 +46,9 @@
 		"dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
 		"mikey179/vfsstream": "^1.6",
 		"phpcompatibility/phpcompatibility-wp": "^2.0",
+		"phpstan/phpstan": "^0.12.3",
```

Comment:
Note: `phpstan/phpstan` and `szepeviktor/phpstan-wordpress` require a minimum of PHP 7.1. The Travis builds are failing for 7.0 and 5.6 due to this.

Reply:
Indeed I noticed that yesterday. We would not need it to run on all builds anyway, do you have an idea on how to work around that?

Reply:
I don't know of a way to do it here. Instead, we'd handle it with Travis, i.e. either in a separate bash script or directly in existing yaml file.

Reply:
For now I removed the need to install phpstan on Travis, since we're not yet ready to have it as part of our CI. In the future we can dedicate a build to run it